src/components/ScholarshipAmountField.test.jsx

```diff
@@ -12,13 +14,19 @@ const formik = {
 
 function renderWithAmountType(type) {
   formik.values.amount.type = type;
-  return render(<ScholarshipAmountField formik={formik} />);
+  return render(
+    <Suspense fallback="loading">
+      <I18nextProvider i18n={i18n}>
+        <ScholarshipAmountField formik={formik} />
+      </I18nextProvider>
+    </Suspense>
```

Comment:
It seems like we need this just about everywhere. We should consider refactoring out this pattern into our [testing.ts](https://github.com/beyondhb1079/s4us/blob/main/src/lib/testing.ts) library and instead import that `render` function here and everywhere (feel free to do this in a separate PR).